trinity/sync/full/state.py

```diff
@@ -82,7 +85,11 @@ def __init__(self,
         self.chaindb = chaindb
         self.peer_pool = peer_pool
         self.root_hash = root_hash
-        self.scheduler = StateSync(root_hash, account_db, self.logger)
+        # We use a LevelDB instance for the nodes cache because a full state download, if run
+        # uninterrupted will visit more than 180M nodes, making an in-memory cache unfeasible.
+        self._nodes_cache_dir = tempfile.TemporaryDirectory(prefix="pyevm-state-sync-cache")
```

Comment:
Just wanted to note that *if* we wanted to re-use this cache we should put it in `$XDG_CACHE_HOME`, though from what I can tell I don't think this is going to be easy to re-use without some fancy orchestration.

Reply:
Yeah, I think reusing this cache is a bad idea as then we'd need to always ensure it is in lockstep with the actual chain DB